### DISCUSSION TITLE
visual mode selections are retained when the window loses focus

### DISCUSSION THREAD

**Discussion by xianyuIC** [NeoVintageous/NeoVintageous#996] [Q&A] [answered]
Hi, the current mode is that even though my current mode is insert mode, when I move my mouse focus away from sublime and go back to sublime, the mode changes to visual mode again. This is very frustrating for users, because when we write code, we often click on other place with the mouse, such as viewing other files, and every time we go back to sublime we change to visual mode.
I know I can set "vintageous_default_mode:" insert ", but then I have to press the esc key every time I only need to browse the file, which is very inconvenient.

Can it be made the same as vim:
When I am now in insert mode, I move my mouse away from sublime and go back to sublime again, still in insert mode.
When I am now in visual mode, I move my mouse away from sublime and go back to sublime again, still in visual mode.

I saw "vintageous_default_mode" in the description, "visual mode selections are retained when the window loses focus", this is too bad, is this convenient to improve?

**Comment by itsmewes** [comment, marked as answer]
Does adding `"vintageous_reset_mode_when_switching_tabs": false,` to the Sublime Settings file (Preferences.sublime-settings) give you the functionality you are looking for?

**Reply by xianyuIC** [reply]
No，after adding it, still this phenomenon.

**Reply by xianyuIC** [reply]
sorry , I have confirmed the wrong issue, I have tried it and it can solve the current problem. Thank you.

**Reply by xianyuIC** [reply]
Thank you very much for your answer. Due to this question, I almost had to give up the sublime, but based on your answer, I successfully solved this problem. This is great!
